connectors/nightcore.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+/* global Connector */
+/* global MetadataFilter */
+
+if (window.frames.length === 0) {
+	// We are in YouTube frame
+	// Overriding artist and track getters because the website contains
+	// better metadata
+
+	let artist = null;
+	let track = null;
+
+	Connector.getArtist = () => artist;
+	Connector.getTrack = () => track;
+	Connector.getArtistTrack = () => {};
+
+	$('video').on('timeupdate', () => {
+		window.parent.postMessage({ sender: 'ws-getArtistTrack' }, '*');
+	});
+
+	window.addEventListener('message', (event) => {
+		if (event.data.sender !== 'ws-artistTrack') {
+			return;
+		}
+		artist = event.data.artist;
+		track = event.data.track;
+	});
+
+	Connector.filter = new MetadataFilter({
+		artist: text => text.replace(/\s*original artist:/i, ''),
+		track: text => text.replace(/^nightcore - /i, ''),
+	});
```

Comment:
The filter doesn't trim song fields, but it should. You can use default `TrimFilter` with appended custom functions (see `MetadataFilter.append` function).